components/Cards/src/MDCCardCollectionCell.m

```diff
@@ -190,11 +206,16 @@ - (void)encodeWithCoder:(NSCoder *)coder {
   [coder encodeObject:_horizontalImageAlignments forKey:MDCCardCellHorizontalImageAlignmentsKey];
   [coder encodeObject:_verticalImageAlignments forKey:MDCCardCellVerticalImageAlignmentsKey];
   [coder encodeObject:_imageTintColors forKey:MDCCardCellImageTintColorsKey];
+  [coder encodeObject:self.layer.shapedBackgroundColor forKey:MDCCardCellBackgroundColorsKey];
 }
 
 - (void)layoutSubviews {
   [super layoutSubviews];
-  self.layer.shadowPath = [self boundingPath].CGPath;
+  if (!self.layer.shapeGenerator) {
+    self.layer.shadowPath = [self boundingPath].CGPath;
+  } else {
+    self.layer.cornerRadius = 0;
```

Comment:
This seems like it might be prone to unexpected behavior. Can we avoid nuking the cornerRadius here?